app/src/main/kotlin/co/netguru/baby/monitor/client/feature/settings/ClientSettingsFragment.kt

```diff
@@ -2,7 +2,6 @@ package co.netguru.baby.monitor.client.feature.settings
 
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/59ecd15255bdd529a391ea0d5072dc37d88e9138/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/settings/ClientSettingsFragment.kt#L3"></span>Imports must be ordered in lexicographic order without any empty lines in-between</td>
    </tr>
  </tbody>
</table>